recovery.go

```diff
@@ -39,6 +56,10 @@ func (rec *Recovery) ServeHTTP(rw http.ResponseWriter, r *http.Request, next htt
 			if rec.PrintStack {
 				fmt.Fprintf(rw, f, err, stack)
 			}
+
+			if rec.ErrorHandlerFunc != nil {
+				rec.ErrorHandlerFunc.HandleError(err, true)
+			}
```

Comment:
With the above switch of the `ErrorHandlerFunc` type, this would of course become:

``` go
        if rec.ErrorHandlerFunc != nil {
            rec.ErrorHandlerFunc(err)
        }
```